commit 546220bf2ef0dc05eb848dacc7e3c956485e357c
Author: sumit <sumitrajdev5842@gmail.com>
Date:   Sat Mar 12 19:33:19 2016 +0530

    rename kernel defconfig

diff --git a/BoardConfig.mk b/BoardConfig.mk
index 76debd3..f553c89 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -21,9 +21,9 @@ TARGET_OTA_ASSERT_DEVICE := kyleve,S7392,GT-S7392,hawaii
 # Kernel
 BOARD_KERNEL_BASE := 0x82000000
 BOARD_KERNEL_PAGESIZE := 4096
-TARGET_KERNEL_CONFIG := bcm21664_hawaii_ss_kyleve_rev01_cm_defconfig
+TARGET_KERNEL_CONFIG := bcm21664_hawaii_ss_kyleve_rev00_defconfig
 TARGET_KERNEL_SOURCE := kernel/samsung/kyleve
-#TARGET_KERNEL_CUSTOM_TOOLCHAIN := arm-eabi-4.6
+TARGET_KERNEL_CUSTOM_TOOLCHAIN := arm-eabi-4.6
 
 # PARTITION SIZE
 BOARD_BOOTIMAGE_PARTITION_SIZE := 8388608
diff --git a/health/Android.mk b/health/Android.mk
index 1317a48..f07d8aa 100644
--- a/health/Android.mk
+++ b/health/Android.mk
@@ -17,6 +17,6 @@ LOCAL_PATH := $(call my-dir)
 
 include $(CLEAR_VARS)
 LOCAL_SRC_FILES := healthd-hawaii.cpp
-LOCAL_MODULE := libhealthd-logands.hawaii
+LOCAL_MODULE := libhealthd-kyleve.hawaii
 LOCAL_C_INCLUDES := system/core/healthd
 include $(BUILD_STATIC_LIBRARY)
